sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/SynchronousReceiveWork.java

```diff
@@ -153,4 +180,11 @@ void startedProcessing() {
     boolean isProcessingStarted() {
         return this.processingStarted;
     }
+
+    @Override
+    public void close() {
+        if (nextMessageSubscriber != null && !nextMessageSubscriber.isDisposed()) {
```

Comment:
is the null check necessary? Isn't this always set in the constructor. You can just call dispose.